index.bs

```diff
@@ -1359,6 +1359,13 @@ whether to use an attribute or a method.
 - Attribute getters must not have any (observable) side effects.
     If you have expected side effects, use a method.
 
+- Attribute getters should not throw exceptions.
+    Exceptions are side effects.
```

Comment:
I'm not sure it's widely accepted that exceptions are side effects. E.g., I'm pretty sure typical side-effect-free functions are still allowed to do input validation.

I can kinda see how they can cause side effects in JavaScript, but only if you have listeners in place that end up modifying state in response to the exception.

Reply:
I think this is beside the point, because exceptions being side effects  is *not* the reason getters should not throw. Getters should behave like regular JS properties, and these do not throw.
Also, typically, invalid state throws when *set*, it's not a good design to allow writes to go through only to realize you have invalid state when someone tries to read it.